src/d3d8/d3d8_batch.h

```diff
@@ -0,0 +1,246 @@
+#pragma once
+
+#include "d3d8_include.h"
+#include "d3d8_buffer.h"
+#include "d3d8_format.h"
+
+#include <vector>
+#include <cstdint>
+#include <climits>
+
+namespace dxvk {
+
+  constexpr size_t            D3DPT_COUNT   = size_t(D3DPT_TRIANGLEFAN) + 1;
```

Comment:
Random things like that in headers should be `static constexpr` or `inline constexpr`.